docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  near-mcp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: near-mcp:latest
+    container_name: near-mcp-server
+    restart: unless-stopped
+    volumes:
+      - ~/.near-keystore:/root/.near-keystore
+    ports:
+      # Expose the SSE server port
+      - "3001:3001"
+    # Override the entrypoint from the Dockerfile
+    entrypoint: []
+    command: ["sh", "-c", "mkdir -p /root/.near-keystore/$${NEAR_NETWORK} && if [ ! -z \"$${NEAR_KEYSTOREDATA}\" ] && [ ! -z \"$${NEAR_NETWORK}\" ] && [ ! -z \"$${NEAR_ACCOUNT_ID}\" ]; then echo \"$${NEAR_KEYSTOREDATA}\" | base64 -d > /root/.near-keystore/$${NEAR_NETWORK}/$${NEAR_ACCOUNT_ID}.json && echo \"Keystore data written to /root/.near-keystore/$${NEAR_NETWORK}/$${NEAR_ACCOUNT_ID}.json\"; fi && cd /app && exec node dist/near-mcp.js run --remote --port 3001"]
```

Comment:
can we extract this into it's own file in `scripts`?

Reply:
Sure, done